frontend/src/metabase/query_builder/components/NativeQueryEditor/CodeMirrorEditor/CodeMirrorEditor.module.css

```diff
@@ -0,0 +1,186 @@
+:global {
+  :local(.editor) {
+    width: 100%;
```

Comment:
`DATE_TRUNC` function is no longer highlighted

[before](https://github.com/user-attachments/assets/632d0c39-bffd-428c-bab0-65be44f2f152) & [after](https://github.com/user-attachments/assets/e794460c-0c06-46b2-840e-67d963676e95)